operators/pkg/controller/common/certificates/http/reconcile.go

```diff
@@ -183,11 +192,11 @@ func ensureInternalSelfSignedCertificateSecretContents(
 	}
 
 	// check if the existing cert should be re-issued
-	if shouldIssueNewHTTPCertificate(es, secret, svcs, ca, rotationParam.RotateBefore) {
+	if shouldIssueNewHTTPCertificate(owner, namer, tls, secret, svcs, ca, rotationParam.RotateBefore) {
 		log.Info(
 			"Issuing new HTTP certificate",
 			"secret", secret.Name,
-			"es", k8s.ExtractNamespacedName(&es),
+			"owner", owner,
```

Comment:
We should log the `namespace` and rename `secret` in `name` or `secret_name` to be in sync with our current effort to standardize the logs (#1168).